src/Parser/VengeanceDemonHunter/CombatLogParser.js

```diff
@@ -127,7 +128,13 @@ class CombatLogParser extends MainCombatLogParser {
 
     results.statistics = [
       <StatisticBox
-        icon={<Icon icon="class_demonhunter" alt="Damage taken" />}
+        icon={<Icon icon="class_demonhunter" alt="Damage done" />}
+        value={`${formatNumber(this.totalDamageDone / this.fightDuration * 1000)} DPS`}
+        label='Damage done'
```

Comment:
Yeah sure and if you're staying alive it's pretty nice to have, but it's a nice-to-have thing and for tanks trying to progress on Mythic bosses it's much more important to focus on staying alive than to contribute a little more damage. A couple thousand DPS gain usually doesn't outweigh staying alive more pulls.